common/supplemental/languageInfo.xml

```diff
@@ -302,6 +302,223 @@
+			<!-- Encompassed by Azerbaijani -->
+			<languageMatch desired="azb" supported="az" distance="10" oneway="true"/>	<!-- South Azerbaijani -->
+			<!-- Encompassed by Estonian -->
+			<languageMatch desired="vro" supported="et" distance="10" oneway="true"/>	<!-- Võro -->
+			<!-- Encompassed by Fulah -->
+			<languageMatch desired="ffm" supported="ff" distance="10" oneway="true"/>	<!-- Maasina Fulfulde -->
+			<languageMatch desired="fub" supported="ff" distance="10" oneway="true"/>	<!-- Adamawa Fulfulde -->
+			<languageMatch desired="fue" supported="ff" distance="10" oneway="true"/>	<!-- Borgu Fulfulde -->
+			<languageMatch desired="fuf" supported="ff" distance="10" oneway="true"/>	<!-- Pular -->
+			<languageMatch desired="fuh" supported="ff" distance="10" oneway="true"/>	<!-- Western Niger Fulfulde -->
+			<languageMatch desired="fui" supported="ff" distance="10" oneway="true"/>	<!-- Bagirmi Fulfulde -->
+			<languageMatch desired="fuq" supported="ff" distance="10" oneway="true"/>	<!-- Central-Eastern Niger Fulfulde -->
+			<languageMatch desired="fuv" supported="ff" distance="10" oneway="true"/>	<!-- Nigerian Fulfulde -->
+			<!-- Encompassed by Guarani -->
+			<languageMatch desired="gnw" supported="gn" distance="10" oneway="true"/>	<!-- Western Bolivian Guaraní -->
+			<languageMatch desired="gui" supported="gn" distance="10" oneway="true"/>	<!-- Eastern Bolivian Guaraní -->
+			<languageMatch desired="gun" supported="gn" distance="10" oneway="true"/>	<!-- Mbyá Guaraní -->
+			<languageMatch desired="nhd" supported="gn" distance="10" oneway="true"/>	<!-- Chiripá -->
+			<!-- Encompassed by Inuktitut -->
+			<languageMatch desired="ikt" supported="iu" distance="10" oneway="true"/>	<!-- Inuinnaqtun -->
+			<!-- Encompassed by Kalenjin -->
+			<languageMatch desired="enb" supported="kln" distance="10" oneway="true"/>	<!-- Markweeta -->
+			<languageMatch desired="eyo" supported="kln" distance="10" oneway="true"/>	<!-- Keiyo -->
+			<languageMatch desired="niq" supported="kln" distance="10" oneway="true"/>	<!-- Nandi -->
+			<languageMatch desired="oki" supported="kln" distance="10" oneway="true"/>	<!-- Okiek -->
+			<languageMatch desired="pko" supported="kln" distance="10" oneway="true"/>	<!-- Pökoot -->
+			<languageMatch desired="sgc" supported="kln" distance="10" oneway="true"/>	<!-- Kipsigis -->
+			<languageMatch desired="tec" supported="kln" distance="10" oneway="true"/>	<!-- Terik -->
+			<languageMatch desired="tuy" supported="kln" distance="10" oneway="true"/>	<!-- Tugen -->
+			<!-- Encompassed by Konkani -->
+			<languageMatch desired="gom" supported="kok" distance="10" oneway="true"/>	<!-- Goan Konkani -->
+			<!-- Encompassed by Kpelle -->
+			<languageMatch desired="gkp" supported="kpe" distance="10" oneway="true"/>	<!-- Guinea Kpelle -->
+			<!-- Encompassed by Kurdish -->
+			<languageMatch desired="ckb" supported="ku" distance="10" oneway="true"/>	<!-- Central Kurdish -->
```

Comment:
See Peter's email question about languages with different default scripts, such as ckb (Arab) vs. ku (Latn). Maybe don't auto-generate close matches for those, and leave it to manual matches when we get explicit information?

Reply:
We could hand-tune any of these we want. Most of them we don't have a lot of information about, so I consider these mappings provisional.

Reply:
However, I dropped 'ku', since the best choice is very unclear.